module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,50 @@
+#include "compression.hpp"
+
+compressed_bitmap compressGrayscale(const bitmap& image) {
+    compressed_bitmap compressedImage{};
+    compressedImage.reserve(height * width);
```

Comment:
when you reserve memory, use `shrink_to_fit` at end

Reply:
Done.